autoload/slime.vim

```diff
@@ -248,6 +249,22 @@ function! s:DtachConfig() abort
   let b:slime_config["socket_path"] = input("dtach socket path: ", b:slime_config["socket_path"])
 endfunction
 
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+" Tabbed
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+function! s:TabbedSend(config, text)
+  call system("xdotool type --delay 0 --window " . b:slime_config["window_id"] . " -- " . shellescape(a:text))
+endfunction
+
+function! s:TabbedConfig() abort
+  if !exists("b:slime_config")
+    let b:slime_config = {"window_id": ""}
+  endif
+  let b:slime_config["window_id"] = trim(system("xwininfo -children -id $XEMBED | grep '     0x' | sed -e's@^ *\\(0x[0-9a-f]*\\) \"\\([^\"]*\\)\".*@\\1 \\2@' | tail -n +2 | dmenu -i -l 5 -p 'REPL window: ' | cut -b 1-9 | xargs printf '%d'"))
```

Comment:
this … is quite a line 😐 

I don't think I can merge/support this. Let me add a comment to discuss alternatives.

Reply:
Sure. I don't know vim script, so what I can do is just shell scripting.